reports/Massachusetts.md

```diff
@@ -18,3 +18,62 @@ id: ma-boston-2
 **Links**
 
 * https://www.youtube.com/watch?v=egvPF75vWOA
+
+## Worcester
+
+### Worcester police officers cover badge numbers during protest | June 1st
+
+id: ma-worcester-3
+
+**Links**
+
+* [Officer with covered badge number](https://twitter.com/sebishop99/status/1267877928391659521/photo/1)
+
+### Riot police throw protestor to the ground | June 1st
+
+A reporter following the protest shows video of the riot police line moving to force protestors back. Then notes that during this, a teenage girl is grabbed, thrown to the ground and surrounded. 
+
+On video, protestors can be heard yelling "You feel big hitting a little woman like that?"
+
+An additional angle provided by the girl's sister shows numerous officers lifting her off the ground and dragging her behind the police line, after which the line closes to obfuscate any view of her.
+
+id: ma-worcester-4
+
+**Links**
+* [Police initially push protestors back](https://twitter.com/sebishop99/status/1267890659484143616)
+* [Reporter says they witnessed a teenage girl slammed to the ground](https://twitter.com/sebishop99/status/1267905527390113798)
+* [Reporter's video showing the aftermath](https://streamable.com/lqxq76)
+* [Additional angle provided by sister](https://twitter.com/sebishop99/status/1267911837808496646)
+
+### Riot police fire on protestors without provocation | June 1st
+
+Officers fire tear gas, rubber bullets and pepper spray BBs at peaceful protestors without warning. Police continue firing as protestors flee.
+
+id: ma-worcester-5
+
+**Links**
+
+* [Riot police firing on fleeing protestors](https://twitter.com/sebishop99/status/1267913459938844673)
+
+### Riot police punch protester | June 1st
+
+id: ma-worcester-6
+
+**Links**
```

Comment:
Needs links!

Reply:
Sorry guys. I collapsed this into the next one and forgot to remove this